android/app/build.gradle

```diff
@@ -196,6 +198,7 @@ dependencies {
     implementation project(':react-native-fast-image')
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation "com.facebook.react:react-native:+"  // From node_modules
+    //implementation 'com.android.support:multidex:1.0.3'
```

Comment:
Is this going to used commonly? If it's not going to be used ever it should just be removed instead of commented

Reply:
Both commented-out lines should be uncommented out if a person's DEX files reach the 64k method reference limit. Right now the master is under that limit but I figure we can have it as a comment in case someone needs more space when they're working in their local repo.